Error on startup with latest version

### What steps will reproduce the problem?

1. Plop latest Towny JAR into `plugins/` directory.
2. Use my configs (see https://github.com/SettleScape/server/tree/master/minecraft/plugins/Towny/)
3. Start server

### What is the expected output?

No error is expected.

### Towny version

v0.99.5.11

### Server version

Paper version git-Paper-411 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT) (Git: 6625db3 on ver/1.17.1)

### Please use Pastebin.com to link the following files

1. Your full server startup from the logs\latest.log: https://gist.github.com/MilesBHuff/8d4e01a754b7a763a98c565e7438431f
2. Your Towny config.yml: https://github.com/SettleScape/server/blob/master/minecraft/plugins/Towny/settings/config.yml
3. Your townyperms.yml: https://github.com/SettleScape/server/blob/master/minecraft/plugins/Towny/settings/townyperms.yml
4. Your log's error:
```
[14:36:23] [Server thread/INFO]: [Towny] Enabling Towny v0.99.5.11
[14:36:23] [Server thread/INFO]: ====================      Towny      ========================
[14:36:23] [Server thread/INFO]: [Towny] Config: 0.98.3.4 applying 1 automatic update...
[14:36:23] [Server thread/INFO]: [Towny] Config: 0.98.3.13 applying 8 automatic updates...
[14:36:24] [Server thread/INFO]: [Towny] Successfully loaded translations for 42 languages.
[14:36:24] [Server thread/INFO]: [Towny] Config: Loaded 9 townblock types: shop, inn, arena, jail, default, bank, wilds, farm, embassy.
[14:36:24] [Server thread/INFO]: [Towny] Database: [Load] flatfile [Save] flatfile
[14:36:24] [Server thread/ERROR]: Error occurred while enabling Towny v0.99.5.11 (Is it up to date?)
java.lang.NoClassDefFoundError: org/apache/commons/lang3/Validate
	at com.palmergames.bukkit.towny.object.Position.deserialize(Position.java:146) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.db.TownyFlatFileSource.loadTown(TownyFlatFileSource.java:845) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.db.TownyDataSource.loadTowns(TownyDataSource.java:200) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.db.TownyDataSource.loadAll(TownyDataSource.java:53) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.TownyUniverse.loadDatabase(TownyUniverse.java:192) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.TownyUniverse.loadAndSaveDatabase(TownyUniverse.java:148) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.Towny.loadFoundation(Towny.java:245) ~[Towny.jar:?]
	at com.palmergames.bukkit.towny.Towny.onEnable(Towny.java:139) ~[Towny.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:561) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:475) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:733) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:317) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1220) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-411]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.Validate
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:151) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[patched_1.17.1.jar:git-Paper-411]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 18 more
[14:36:24] [Server thread/INFO]: [Towny] Disabling Towny v0.99.5.11
[14:36:24] [Server thread/INFO]: ==============================================================
[14:36:24] [Server thread/INFO]: [Towny] Finishing File IO Tasks...
[14:36:24] [Server thread/INFO]: [Towny] Finishing Universe Tasks...
[14:36:24] [Server thread/INFO]: [Towny] Version: 0.99.5.11 - Plugin Disabled
```

(optional) If this is to do with permissions,
5. Your permissions file: https://github.com/SettleScape/server/tree/master/minecraft/plugins/LuckPerms/yaml-storage/groups

(optional) If this is to do with chat,
6. Your chatconfig.yml: https://github.com/SettleScape/server/blob/master/minecraft/plugins/Towny/settings/ChatConfig.yml
7. Your channels.yml: https://github.com/SettleScape/server/blob/master/minecraft/plugins/Towny/settings/Channels.yml

--- a/Towny/src/main/java/com/palmergames/bukkit/towny/object/Position.java
+++ b/Towny/src/main/java/com/palmergames/bukkit/towny/object/Position.java
@@ -1,7 +1,7 @@
 package com.palmergames.bukkit.towny.object;
 
+import com.google.common.base.Preconditions;
 import com.palmergames.bukkit.towny.TownyAPI;
-import org.apache.commons.lang3.Validate;
 import org.bukkit.Location;
 import org.bukkit.World;
 import org.bukkit.util.NumberConversions;
@@ -71,7 +71,7 @@ public class Position {
 	}
 	
 	public static Position positionOf(@NotNull TownyWorld world, double x, double y, double z, float pitch, float yaw) {
-		Validate.notNull(world, "world cannot be null");
+		Preconditions.checkNotNull(world, "world cannot be null");
 		return new Position(world, x, y, z, pitch, yaw);
 	}
 	
@@ -86,7 +86,7 @@ public class Position {
 	}
 	
 	public static Position ofLocation(@NotNull Location location) {
-		Validate.notNull(location, "location cannot be null");
+		Preconditions.checkNotNull(location, "location cannot be null");
 		
 		final World bukkitWorld = location.getWorld();
 		if (bukkitWorld == null)
@@ -143,7 +143,7 @@ public class Position {
 	}
 	
 	public static Position deserialize(@NotNull final String[] data) throws IllegalArgumentException {
-		Validate.notNull(data, "data cannot be null");
+		Preconditions.checkNotNull(data, "data cannot be null");
 		
 		final TownyWorld world = TownyAPI.getInstance().getTownyWorld(data[0]);
 		

